### PR TITLE
Adding webpack

### PR DESCRIPTION
**Pull request by Felipe-Perez-Ferraro**
In this repository i added webpack by following the instructions gaven by Microverse

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -248,44 +181,26 @@ Feel free to check the [issues page](../../issues/).
...
-
-  - [Answer_2]
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (Efiamotu-1) [inline README.md:203]: - [ ] Your license link is broken as currently when you click on it, it doesn't display your license file rather it displays a 404 error page shown below please kindly fix this 👍 
mit

File: README.md [hunk 2]
```diff
@@ -46,191 +43,127 @@ For more information on the importance of a professional README for your reposit
...
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
 
-- [Live Demo Link](https://google.com)
+- [Live Demo Link](https://feliperto.github.io/Awesome-Books-ES6/)
```
Comment (Efiamotu-1) [inline README.md]: - [ ] Your live demo still shows your previous project which is `Awesome Books` please kindly provide the correct project for your live demo 👍

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+// const removeTask = () => {
+//   const completeTask = document.querySelectorAll('.marked')
+//   tasks.forEach((it) => {
+//     if (it.completed === true) {
+//       completeTask.forEach((el) => {
+//         el.remove()
+//       });
+//     }
+//   });
+// }
```
Comment (Efiamotu-1) [inline src/index.js]: - [ ] Leaving commented code is highly unprofessional, I kindly suggest you remove them 👍

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+let tasks = [];
+
+const setTask = () => {
+  const task = {
+    id: Date.now(),
+    description: addInput.value,
+    completed: false,
+  };
+  tasks = [...tasks, task];
+};
```
Comment (Efiamotu-1) [inline src/index.js]: - [ ] Your project is missing the project requirements as shown in the image below
req
- [ ] You are required to create an array with a list of todo tasks, see the code below for example
```javascript
const tasks = [
  {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
];
```
- [ ] After creating the tasks you are to iterate over the tasks, see the code below for guidance

```javascript
const populateTasks = (task, parentElement) => {
  const { description, completed, index: id } = task;
  const innerHTML = `
         <p>${description}</p>
         <p>${completed ? 'completed' : 'not completed'}</p>`;
  const newTodoListElement = document.createElement('li');
  Object.assign(newTodoListElement, { innerHTML, id });
  parentElement.appendChild(newTodoListElement);
};
```
- [ ] Once you have achieved the two steps described above when the page is loaded the todos created in the array of objects would be displayed on your webpage. Please i kindly suggest you re-read the project requirement and see the examples and implement accordingly 👍

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @Feliperto , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ Good job on achieving the project design 👍 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — APPROVED** [review]
# Hi @Feliperto,  👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights

- [ ]  No Linters errors 🔥
- [ ]  Great job working on the functionality💯
- [ ]  Kudos for working on the requested changes🔥
- [ ]  Followed GitHub Flow ⭐🚀


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**